src/libraries/Microsoft.Extensions.Logging/tests/Common/LoggerFactoryTest.cs

```diff
@@ -92,6 +94,45 @@ public void Dispose_ThrowException_SwallowsException()
                 .Verify(p => p.Dispose(), Times.Once());
         }
 
+        [Fact]
```

Comment:
This should probably be a theory that makes sure each combination of flags prints the appropriate things. To avoid 2^n we can just hard code some combinations.

Reply:
That is a good idea. Actually I have added all combinations. Just to have me feel more comfortable :-)